src/main/scala/vectorpipe/vectortile/Pipeline.scala

```diff
@@ -148,5 +148,5 @@ trait Pipeline {
    * See [[geotrellis.vectortile.Value]] for details.
    */
   def pack(row: Row, zoom: Int): VectorTileFeature[Geometry] =
-    Feature(Geometry(row.getAs[jts.Geometry](geometryColumn)), Map.empty)
+    Feature(row.getAs[jts.Geometry](geometryColumn), Map.empty)
```

Comment:
Prefer here to not import JTS, import all of `vector` and drop these `jts.` prefixes.  This comment will surely apply at other points in this PR.  `jts.geom` should basically not be imported unless you're working with coordinates.